┌─[ symbianflo @ symbianflo ] - [ Mandrivausers.ro ] 
└─[ MRB:aint-no-shit $]: diff -u build/functions build/functions.new
====================================================================
--- build/functions     2014-03-25 04:18:47.000000000 +0100
+++ build/functions.new 2014-05-16 23:18:15.290442006 +0200
@@ -456,7 +456,7 @@
 check_gogglesmm()
 {
   add_required_package "sqlite3 >= 3.6.19" "sqlite"
-  add_required_package "taglib >= 1.9.0" "taglib"
+  add_required_package "taglib >= 1.8" "taglib"
   if [ $OPTION_GMM_DBUS -eq 1 ] ; then
     add_package "dbus-1 >= 1.0" "dbus"
     OPTION_GMM_DBUS=$?
